src/app/pages/BlocksPage/index.tsx

```diff
@@ -52,26 +72,51 @@ export const BlocksPage: FC = () => {
             },
           }
         },
+        keepPreviousData: tableView === TableView.Vertical,
       },
     },
   )
 
   return (
-    <PageLayout>
+    <PageLayout
+      mobileFooterAction={<LoadMoreButton pagination={pagination} isLoading={blocksQuery.isLoading} />}
```

Comment:
This appears in horizontal mode too: 
![image](https://user-images.githubusercontent.com/3758846/228393507-aa12bd46-13c3-4020-9583-539c3438373f.png)